Assets/HoloToolkit/Utilities/Scripts/FixedAngularSize.cs

```diff
@@ -12,20 +12,29 @@ namespace HoloToolkit.Unity
     /// </summary>
     public class FixedAngularSize : MonoBehaviour
     {
+        /// <summary>
+        /// Off sets the scale ratio so that text does not scale down so much. (Set to zero for linear scaling)
+        /// </summary>
+        public float OverrideSizeRatio = 0;
         // The ratio between the transform's local scale and its starting
         // distance from the camera.
-        private Vector3 defaultSizeRatios;
+        private float startingDistance;
+        private Vector3 startingScale;
 
         void Start()
         {
             // Calculate the XYZ ratios for the transform's localScale over its
             // initial distance from the camera.
-            float startingDistance = Vector3.Distance(Camera.main.transform.position, transform.position);
-            if (startingDistance > 0.0f)
+            startingDistance = Vector3.Distance(Camera.main.transform.position, transform.position);
+            startingScale = transform.localScale;
+
+            if (OverrideSizeRatio == 0)
             {
-                defaultSizeRatios = transform.localScale / startingDistance;
+                // set to a linear scale ratio
+                OverrideSizeRatio = 1 / startingDistance;
             }
-            else
+
+            if (startingDistance < 0.0f)
```

Comment:
Regarding the comment below about hitting a DivideByZero error later on, looking back in time, it appears that comment is out of date. In an earlier implementation of this class, there was a division by startingDistance made during Update() which is no longer the case.

Yes, line 34 needs to be protected against DivideByZero.